internal/store/model/assessment.go

```diff
@@ -23,10 +23,22 @@ type Assessment struct {
 	Snapshots      []Snapshot `gorm:"foreignKey:AssessmentID;references:ID;constraint:OnDelete:CASCADE;"`
 }
 
+type SnapshotStatus string
+
+const (
+	SnapshotStatusPending    SnapshotStatus = "pending"
+	SnapshotStatusParsing    SnapshotStatus = "parsing"
+	SnapshotStatusValidating SnapshotStatus = "validating"
+	SnapshotStatusReady      SnapshotStatus = "ready"
+	SnapshotStatusFailed     SnapshotStatus = "failed"
+)
+
 type Snapshot struct {
 	ID           uint                      `gorm:"primaryKey;autoIncrement"`
 	CreatedAt    time.Time                 `gorm:"not null;default:now()"`
-	Inventory    *JSONField[api.Inventory] `gorm:"type:jsonb;not null"`
+	Status       SnapshotStatus            `gorm:"type:varchar(20);default:'pending'"`
```

Comment:
Nice to have: UpdatedAt could be useful to provide a nice feedback to the user on the processing time.